apps/public-reference/pages/applications/preferences/select.tsx

```diff
@@ -0,0 +1,189 @@
+/*
+4.1 Preferences Introduction
+Instructions on how preferences work and their value
+*/
+import Link from "next/link"
+import Router from "next/router"
+import { Button, ErrorMessage, FormCard, ProgressNav, t } from "@bloom-housing/ui-components"
+import FormsLayout from "../../../layouts/forms"
+import { useForm } from "react-hook-form"
+import { AppSubmissionContext } from "../../../lib/AppSubmissionContext"
+import ApplicationConductor from "../../../lib/ApplicationConductor"
+import FormStep from "../../../src/forms/applications/FormStep"
+import { useContext, useMemo, useState } from "react"
+
+export default () => {
+  const { conductor, application, listing } = useContext(AppSubmissionContext)
+  const currentPageStep = 4
+
+  const preferenceOptions = ["liveIn", "workIn"]
+
+  /* Form Handler */
+  const { getValues, register, handleSubmit, errors, setValue, trigger } = useForm()
+  const onSubmit = (data) => {
+    new FormStep(conductor).save({ preferences: data })
+    if (data.none) {
+      conductor.routeToNextOrReturnUrl("/applications/preferences/general")
+    } else {
+      conductor.completeStep(4)
+      conductor.sync()
+      conductor.routeToNextOrReturnUrl("/applications/review/demographics")
+    }
+  }
+
+  return (
+    <FormsLayout>
+      <FormCard header="LISTING">
+        <ProgressNav
+          currentPageStep={currentPageStep}
+          completedSteps={application.completedStep}
+          labels={["You", "Household", "Income", "Preferences", "Review"]}
+        />
+      </FormCard>
+
+      <FormCard>
+        <p className="form-card__back">
+          <strong>
+            <Link href="/applications/financial/income">
+              <a>{t("t.back")}</a>
+            </Link>
+          </strong>
+        </p>
+
+        <div className="form-card__lead border-b">
+          <h2 className="form-card__title is-borderless">{t("application.preferences.title")}</h2>
+
+          <p className="field-note mt-5">{t("application.preferences.preamble")}</p>
+        </div>
+
+        <form className="" onSubmit={handleSubmit(onSubmit)}>
+          <div className="form-card__group px-0 pb-3">
+            <p className="field-note">{t("application.preferences.selectBelow")}</p>
+          </div>
+
+          {preferenceOptions.map((option) => {
+            const [showMore, setShowMore] = useState(false)
+            return (
+              <div className="form-card__group px-0 border-b">
```

Comment:
This needs a `key` property - anytime you use `map` in JSX, you _need_ to provide a `key` (you almost certainly have console warnings about this if you look).